x-pack/platform/test/spaces_api_integration/access_control_objects/apis/spaces/feature_disabled.ts

```diff
@@ -0,0 +1,939 @@
+
+  const loginAsNotObjectOwner = (username: string, password: string) => login(username, password);
+
+  // This test suite relies on the access_control_test_plugin, but the feature flag is explicitly disabled
+  // in the congig. This means that ACCESS_CONTROL_TYPE is still registered as supporting access control,
+  // however, the feature is disabled and the type will not support access control in practice. These tests
+  // aim to validate that the object type still behaves as expected - like any other object that does not
+  // support access control.
+  describe('access control saved objects - feature disabled', () => {
+    before(async () => {
+      await security.testUser.setRoles(['kibana_savedobjects_editor']);
+      await createSimpleUser();
+    });
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    describe('#create', () => {
+      it('rejects creating a write-restricted object', async () => {
+        const { cookie: adminCookie } = await loginAsKibanaAdmin();
+        const response = await supertestWithoutAuth
+          .post('/access_control_objects/create')
+          .set('kbn-xsrf', 'true')
+          .set('cookie', adminCookie.cookieString())
+          .send({ type: ACCESS_CONTROL_TYPE, isWriteRestricted: true })
+          .expect(400);
+
+        expect(response.body).to.have.property('error', 'Bad Request');
+        expect(response.body.message).to.contain(
+          `Cannot create a saved object of type ${ACCESS_CONTROL_TYPE} with an access mode because the type does not support access control: Bad Request`
+        );
+      });
+
+      it('allows creating an object without access control metadata', async () => {
+        const { cookie: adminCookie } = await loginAsKibanaAdmin();
+        const response = await supertestWithoutAuth
+          .post('/access_control_objects/create')
+          .set('kbn-xsrf', 'xxxxx')
+          .set('cookie', adminCookie.cookieString())
+          .send({ type: ACCESS_CONTROL_TYPE })
+          .expect(200);
+        expect(response.body).not.to.have.property('accessControl');
+        expect(response.body).to.have.property('type');
+        const { type } = response.body;
+        expect(type).to.be(ACCESS_CONTROL_TYPE);
+      });
+
+      it('allows creating an object when there is no active user profile', async () => {
+        const response = await supertest
+          .post('/access_control_objects/create')
+          .set('kbn-xsrf', 'true')
+          .send({ type: ACCESS_CONTROL_TYPE })
+          .expect(200);
+        expect(response.body).not.to.have.property('accessControl');
+        expect(response.body).to.have.property('type');
+        const { type } = response.body;
+        expect(type).to.be(ACCESS_CONTROL_TYPE);
+      });
+
+      // Note: would be better to test this against an object with access control metadata and
+      // verify that it makes no difference to the outcome
```

Comment:
Attn @elastic/kibana-core - we tried to set this up using the esArchiver to create SO's with access control metadata, but we could not get it to work. IIRC the raw documents existed, but they were not recognized as SO's (we could not get or find them from the SOR). Do you have any suggestions for how we could test this case?